### PR TITLE
Add Amazon Kindle Fire HDX 7 (thor)

### PR DESCRIPTION
**Pull request by steamp0rt**

### CODE REVIEW

**Comment by jbruechert** [comment]
Thank you for the pull request!
Please remove the revision from the los and them remote, since 14.1 is already the default. Please also adapt the indentation to how it's done in the other manifests.

You will have to fork the kernel repository to add your modified defconfig. Please create a new halium-7.1 branch on the repository and switch it to the hal remote.

File: manifests/amazon-thor.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+            revision="refs/heads/cm-14.1" />
+
+    <remote name="ggow"
+            fetch="https://github.com/ggow"
+            revision="refs/heads/or-8.1" />
+
+  <project path="device/amazon/thor" name="android_device_amazon_thor" remote="los" />
+  <project path="device/amazon/hdx-common" name="android_device_amazon_hdx-common" remote="los" />
+
+  <project path="kernel/amazon/hdx-common" name="kernel_phoenix_hdx-common" remote="ggow" />
```
Comment (lnjX) [inline manifests/amazon-thor.xml]: Please fork the kernel, create a halium-7.1 branch and publish your kernel config there.

File: manifests/amazon-thor.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            revision="refs/heads/cm-14.1" />
+
+    <remote name="ggow"
+            fetch="https://github.com/ggow"
+            revision="refs/heads/or-8.1" />
+
+    <project path="device/amazon/thor" name="android_device_amazon_thor" remote="los" />
+    <project path="device/amazon/hdx-common" name="android_device_amazon_hdx-common" remote="los" />
+
+    <project path="kernel/amazon/hdx-common" name="kernel_phoenix_hdx-common" remote="ggow" />
```
Comment (lnjX) [inline manifests/amazon-thor.xml]: Please fork the kernel, create a halium-7.1 branch and publish your kernel config there.

Comment (ghost) [reply manifests/amazon-thor.xml]: Done

**Comment by ghost** [comment]
Thanks for fixing my crappy xml skills

**Pull request merged**